package.json

```diff
@@ -47,11 +47,15 @@
     "@microsoft/api-documenter": "^7.13.76",
     "@microsoft/api-extractor": "^7.19.2",
     "@types/jest": "^26.0.15",
-    "@typescript-eslint/eslint-plugin": "^4.6.0",
-    "@typescript-eslint/parser": "^4.6.0",
+    "@typescript-eslint/eslint-plugin": "^5.16.0",
+    "@typescript-eslint/parser": "^5.16.0",
```

Comment:
Do we still need these as top-level dependencies? I'm wondering if the shared ESLint config dep takes care of them.

Reply:
I think Oliver tried removing them and it caused tests to fail, but I'll look into it more to make sure

Reply:
We should be able to remove all of these dependencies because npm 7 and later automatically installs peer dependencies. But, we run tests using node 12 and 14, which use npm 6, so the peer dependencies aren't installed and those tests fail. So, if we want to continue testing on the older node versions, I think these need to be specified as top-level dependencies